extractors/python_functions/window_search/code_snippet.md

```diff
@@ -1,10 +1,12 @@
 ```python
-YOUR_WINDOW = 4 # choose any window size here
-YOUR_LABEL = "PERSON"
-YOUR_ATTRIBUTE = "details" # choose any available attribute here
-LOOKUP_VALUES = ["join"] # this could also be coming from a knowledge base via `import knowledge`
+from typing import List
 
-def window_cue_search(doc):
+YOUR_WINDOW: int = 4 # choose any window size here
+YOUR_LABEL: str = "PERSON"
+YOUR_ATTRIBUTE: str = "text" # choose any available attribute here
+LOOKUP_VALUES: List[str] = ["join"] # this could also be coming from a knowledge base via `import knowledge`
```

Comment:
YOUR_ missing 
also knowledge / list decision (maybe two version?)

Reply:
YOUR fixed, decision for multiple versions still open

Reply:
Would it be possible to do both in one version?

Reply:
yes, it'S the fix i posted with gazetteer --> ill implement it